DashWallet/Sources/UI/CrowdNode/Getting Started/GettingStartedViewController.swift

```diff
@@ -105,13 +109,17 @@ extension GettingStartedViewController {
     private func configureHierarchy() {
         view.backgroundColor = UIColor.dw_secondaryBackground()
 
-
         logoWrapper.layer.dw_applyShadow(with: .dw_shadow(), alpha: 0.05, x: 0, y: 0, blur: 10)
         newAccountButton.layer.dw_applyShadow(with: .dw_shadow(), alpha: 0.1, x: 0, y: 0, blur: 10)
         linkAccountButton.layer.dw_applyShadow(with: .dw_shadow(), alpha: 0.1, x: 0, y: 0, blur: 10)
 
         let minimumDash = CrowdNode.minimumRequiredDash.formattedDashAmount
-        minimumBalanceLable.text = String.localizedStringWithFormat(NSLocalizedString("You need at least %@ on your Dash Wallet", comment: "CrowdNode"), minimumDash)
+        minimumBalanceLabel.text = String.localizedStringWithFormat(NSLocalizedString("You need at least %@ on your Dash Wallet", comment: "CrowdNode"), minimumDash)
+        titleLabel.text = NSLocalizedString("Getting started with CrowdNode", comment: "CrowdNode")
+        subtitleLabel.text = NSLocalizedString("To start staking, create an account on CrowdNode or connect to an existing one.", comment: "CrowdNode")
+        newAccountTitle.text = NSLocalizedString("New CrowdNode Account", comment: "CrowdNode")
+        passphraseLabel.text = NSLocalizedString("You have to backup your passphrase in Dash Wallet", comment: "CrowdNode")
+        linkAccountLabel.text = NSLocalizedString("Link Existing Account", comment: "CrowdNode")
```

Comment:
Not sure if it's a good idea to duplicate all strings from storyboards in the code. For CrowdNode, this will have to do.

We only have one file that transifex pulls from for translations, so perhaps if we have another big feature with a lot of text, it's better to localize it and add the file to transifex config.

Reply:
I feel like it's ok for now